.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Add Tags to Release
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: get the release version
+      id: get_version
+      run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+    - name: tag with release version
+      uses: tinact/docker.image-retag@1.0.2
+      with:
+        image_name: rht-labs/omp-git-api
+        image_old_tag: ${{ github.sha }}
+        image_new_tag: ${{ steps.get_version.outputs.VERSION }}
```

Comment:
aren't we able to get the version from ${{ github.ref }}?

From doc `The branch or tag ref that triggered the workflow run.` https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions

Reply:
${{ github.ref }} actually resolves as /refs/tags/{tag}. or it was when this action was running.  i found post that showed this is how they worked around it.